objax/util/image.py

```diff
@@ -12,17 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__all__ = ['nchw', 'nhwc', 'normalize_to_uint8', 'normalize_to_unit_float', 'to_png']
+__all__ = ['from_file', 'image_grid', 'nchw', 'nhwc', 'normalize_to_uint8', 'normalize_to_unit_float', 'to_png']
 
 import io
-from typing import Union
+from typing import Union, BinaryIO, IO
 
+import jax.numpy as jn
 import numpy as np
 from PIL import Image
 
 from objax.typing import JaxArray
 
 
+def from_file(file: Union[str, IO[BinaryIO]]) -> np.ndarray:
+    """Read an image from a file, convert it RGB and return it as an array.
+
+    Args:
+        file: filename or python file handle of the input file.
+
+    Return:
+        3D numpy array (C, H, W) normalized with normalize_to_unit_float.
+    """
+    image = np.asarray(Image.open(file).convert('RGB'))
+    return normalize_to_unit_float(image.transpose((2, 0, 1)))
+
+
+def image_grid(image: np.ndarray) -> np.ndarray:
+    """Take an image grid (nh, nw, gw, c, h, w) and export it to an image (c, nh * h, nh * w)."""
```

Comment:
What is `gw`? From the code of the function it looks like input image should have 5 ndims, not 6.

I assume that this the main purpose of this function for visualisation (take a batch (nh, nw) of images and convert it into image grid. In this case maybe better explanation would be something like following:

```Rearrange array of images (nh, hw, c, h, w) into image grid on single image.```

i.e. basically emphasize that input is batch/matrix/array of images and output is single image where source images are arranged as a grid.

Reply:
Thanks, done.